pkg/cluster/cluster.go

```diff
@@ -65,6 +65,9 @@ type Spec struct {
 
 	// Simulate creating a cluster but don't actually create it
 	DryRun *bool
+
+	// Disable SCP checks in the installer by setting credentials mode as mint
+	DisableSCPChecks *bool
```

Comment:
Question - Will it be nicer if instead having a new field in the cluster, we set the given argument in CCS.DisableSCPChecks? Can we do that at all?

Reply:
We could, but it's probably unnecessary, this is a temporary cluster object only used to capture the command-line flags. What do you think?

Reply:
Oh..I see. /lgtm then.

Reply:
lgtm